src/main/scala/nodes/nlp/POSTagger.scala

```diff
@@ -0,0 +1,46 @@
+package nodes.nlp
+
+import epic.sequences.{CRF, TaggedSequence}
+import epic.trees.AnnotatedLabel
+import workflow.Transformer
+import org.apache.spark.broadcast.Broadcast
+
+/**
+  * Transformer that uses Epic to part-of-speech tag a sequence of words.
+  * It is recommended to use a Tokenizer before applying the POS Tagger,
+  * assuming we get a list of strings as input.
+  *
+  * Note: Epic model is kind of heavy (~300MB) so it may take some time to load
+  * TODO: Test the model on a Spark cluster w/ and w/o broadcast variable for benchmark
+  *
+  * Here's an example:
+  * {{{
+  *   val model = epic.models.PosTagSelector.loadTagger("en").get
+  *   val POSTagger = POSTagger(model).apply(data)
+  * }}}
+  *
+  * @param model The POS Tagger model loaded from the Epic library
+  */
+case class POSTagger(model: CRF[AnnotatedLabel, String])
+  extends Transformer[Array[String], TaggedSequence[AnnotatedLabel, String]] {
```

Comment:
In general I get a little nervous about exposing classes from other-libraries at user-facing interfaces, but this class from the epic API has a reasonably simple definition and has a bunch of the functionality we're after, so I'm ok with this for now. If it becomes a problem in the future, providing a Keystone-specific version of the interface that's API-compatible shouldn't be too hard.